EfEnumToLookup/LookupGenerator/EnumReference.cs

```diff
@@ -10,14 +10,15 @@ namespace EfEnumToLookup.LookupGenerator
 	[DebuggerDisplay("{DebuggerDisplay,nq}")]
 	internal class EnumReference
 	{
-		public string ReferencingTable { get; set; }
+		public string ReferencingTable { get; set; }    
 		public string ReferencingField { get; set; }
+		public string ReferencingSchema { get; set; }
 		public Type EnumType { get; set; }
 
 		// ReSharper disable once UnusedMember.Local
 		private string DebuggerDisplay
 		{
-			get { return string.Format("EnumReference: {0}.{1} ({2})", ReferencingTable, ReferencingField, EnumType.Name); }
+			get { return string.Format("EnumReference: {3}.{0}.{1} ({2})", ReferencingTable, ReferencingField, EnumType.Name, ReferencingSchema); }
```

Comment:
re-order these for readability, or better still convert to the new interpolated string format. I might do a separate commit to use interpolation everywhere